instrumentation/grizzly-2.0/src/main/java/io/opentelemetry/auto/instrumentation/grizzly/HttpCodecFilterAdvice.java

```diff
@@ -43,6 +43,8 @@ public static void onExit(
     HttpRequestPacket httpRequest = (HttpRequestPacket) httpHeader;
     Span span = TRACER.startSpan(httpRequest, httpRequest, method);
 
-    TRACER.startScope(span, ctx);
+    // We don't actually want to attach new context to this thread, as actual request will continue
+    // on some other thread. But we do want to attach that new context to FilterChainContext
+    TRACER.startScope(span, ctx).close();
```

Comment:
just curious, is this related to this PR, or a fix for #845?

Reply:
This test fail on me locally, so I fixed it. It probable is related to that...